### ISSUE TITLE
convert to jekyl

### ISSUE THREAD

**Issue by wardi** [wardi/jsonlines#8]
editing html is tiresome

**Comment by sp4ce** [comment]
I love Jekyll, I would totally be happy the pickup this issue. I am going to assign it to myself, as I will have some freetime coming up.

**Comment by sp4ce** [comment]
@wardi I was looking at that [deploying-with-cloudflare-pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-jekyll-site/#deploying-with-cloudflare-pages) and I don't have access to the cloud fare configuration, but could you help me confirm is suitably configured? I created a draft pull request https://github.com/wardi/jsonlines/pull/71 and it deployed to https://8-convert-to-jekyl.jsonlines.pages.dev but we would need to use the `_site` folder instead.

**Comment by wardi** [comment]
the logs say "No build command specified" and "Uploaded 0 files", what changes need to be made on the cloudflare configuration?

```
10:43:36.807 | Cloning repository...
-- | --
10:43:37.637 | From https://github.com/wardi/jsonlines
10:43:37.638 | * branch            0c63e447cbaad42565073a0e36a672748e7ea267 -> FETCH_HEAD
10:43:37.638 |  
10:43:37.703 | HEAD is now at 0c63e44 Merge branch 'gh-pages' into 8-convert-to-jekyl
10:43:37.703 |  
10:43:37.810 |  
10:43:37.839 | Success: Finished cloning repository files
10:43:38.428 | No build command specified. Skipping build step.
10:43:38.428 | Note: No functions dir at /functions found. Skipping.
10:43:38.428 | Validating asset output directory
10:43:39.218 | Deploying your site to Cloudflare's global network...
10:43:41.913 | Uploading... (44/44)
10:43:41.914 | ✨ Success! Uploaded 0 files (44 already uploaded) (0.72 sec)
10:43:41.914 |  
10:43:42.619 | ✨ Upload complete!
10:43:47.000 | Success: Assets published!
10:43:47.914 | Success: Your site was deployed!
```

**Comment by wardi** [comment]
Might be easier to fork this repo and set up a new free cloudflare pages site as a test.

**Comment by sp4ce** [comment]
Yes I can do that, but the change we need to make on the cloudfare configuration are described [here](https://developers.cloudflare.com/pages/framework-guides/deploy-a-jekyll-site/#deploying-with-cloudflare-pages) basically it seems it's in `Set up builds and deployments` and then put those values:

![Screenshot 2023-06-10 at 11 56 50](https://github.com/wardi/jsonlines/assets/497642/52e9ce37-78ae-48c7-9443-46dd570a3e6a)

But I'll have a look at forking and configure it on my side as well if needed.

**Comment by sp4ce** [comment]
@wardi, yup I fixed the [branch](https://github.com/wardi/jsonlines/pull/71/files) after testing on cloudfare, it's very simple configuration. Pasting it here 

![Screenshot 2023-06-11 at 11 50 55](https://github.com/wardi/jsonlines/assets/497642/c63b9154-a5be-4578-9f64-1d22718f2299)

The pull request is still in early stages anyway and I will try some complete the basic scaffolding for jekyll. However we could also put it in place, with like no special processing (it copies the files into `_site` and then serve them) and then iteratively "jekyllized" the website later. What do you think?

**Comment by sp4ce** [comment]
@wardi Sorry I did not mean to close it and merge my PR, but I realized I had the wrong tracking branch on my local :/ I looked it up online on jsonlines.org and all seems good. Let me know if you want that I revert everything, or if I need to change something based on the PR above. Sorry again.

**Comment by wardi** [comment]
No worries. Site still seems to look correct.

Is the automated workflow set up so that changes in the .md files will be reflected?
